app/src/main/java/com/hedvig/app/feature/offer/ui/changestartdate/ChangeDateBottomSheetViewModel.kt

```diff
@@ -17,10 +21,14 @@ class ChangeDateBottomSheetViewModel(
 ) : ViewModel() {
 
     private val _viewState = MutableStateFlow<ViewState>(ViewState.Inceptions(data.inceptions))
-    val viewState: StateFlow<ViewState> = _viewState
+    val viewState: StateFlow<ViewState> = _viewState.asStateFlow()
 
     private val selectedDates = mutableMapOf<String, LocalDate>()
 
+    fun shouldOpenDatePicker(): Boolean {
+        return _viewState.value !is ViewState.Loading
+    }
```

Comment:
```suggestion
    fun shouldOpenDatePicker() = viewState.value !is ViewState.Loading
```

Two-part suggestion:

1. When reading, it's probably a good idea to use the read-handle instead of the write-handle. Not super-important when internally in the `ViewModel`, but one could for example imagine that we `map` on the `viewState` at a later stage, and then we would risk reading the 'incorrect' value here.
2. I'm partial to the functional syntax for functions when there is only one statement in a function, but this is personal preference. I like how concise it is, it makes it more readable to me 🙂

Reply:
I agree with both of these. An oversight by me, fixed now.